### PR TITLE
Update map.R: issue #54

### PR DESCRIPTION
**Pull request by sumprain**
Another way to solve the issue #54. earlier way is done by lionel.

### CODE REVIEW

**Comment by lionel-** [comment]
your solution is indeed better.

@hadley, if you think the modifications I did to `partial()` in #55 are useful for other purposes, I'll update it to remove the `map_n()` fix so both PR can be merged.

**Comment by hadley** [comment]
Could you please merge/rebase?

**Comment by sumprain** [comment]
Hadley, I dont understand what to do? Please clarify.

**Comment by hadley** [comment]
Have a look at the instructions at http://r-pkgs.had.co.nz/git.html#pr-make

**Comment by sumprain** [comment]
Hadley, thanks a lot for the help. I think I have done it successfully.

File: tests/testthat/test-map.R [hunk 1]
```diff
@@ -20,17 +20,13 @@ test_that("map2() and map3() return a data frame when given one", {
   expect_is(out_map3, "data.frame")
 })
 
-test_that("map_n() is not vectorized over additional arguments (see #54)", {
```
Comment (hadley) [inline tests/testthat/test-map.R]: Did you accidentally delete these tests?

Comment (sumprain) [inline tests/testthat/test-map.R]: I am afraid yes.

**Pull request closed without merging**